swift/Sources/FlatBuffers/ByteBuffer.swift

```diff
@@ -352,7 +354,14 @@ public struct ByteBuffer {
   ///   - position: the index of the object in the buffer
   @inline(__always)
   public func read<T>(def: T.Type, position: Int) -> T {
```

Comment:
What do you think of doing:

```swift
  public func read<T>(def: T.Type, position: Int, unalignedSupported: Bool = false) -> T {
```

instead of the `readUnalignedBuffers` property?

Reply:
Yeah that an option, however from my point of view it seems to be less safe. Since someone might forget to pass it, and then crash their app doing so.

Reply:
yeah, I agree that `read` is not good place to add this flag. @mustiikhalil what do you think about adding this flag to the `init`? I see there is only one init function of ByteBuffer which assumes using unowned memory. What about to add the flag here:
```
public init(
    assumingMemoryBound memory: UnsafeMutableRawPointer,
    capacity: Int,
    unalignedRead: Bool = false)
  {
...
```
or make a separate `init`:
```
public init(
    assumingUnlignedMemory memory: UnsafeMutableRawPointer,
    capacity: Int)
  {
...
```

In the rest - great stuff! Thank you for supporting this!

Reply:
Should be addressed now